core/src/test/java/io/grpc/ServiceProvidersTestAbstractProvider.java

```diff
@@ -0,0 +1,12 @@
+package io.grpc;
```

Comment:
This needs a copyright. I think that's the cause of the complaint from checkstyle on Travis